README.md

```diff
@@ -113,19 +113,64 @@ There is a provided docker-compose file (docker-compose-confluent-cloud.yml) tha
 ## Note
 This has only been tested with Confluent Cloud but technically all SASL authenticated Kafka brokers can be reached using this method.
 
-# Run Script
-The run.sh script can be utilized to run the PPM manually.
+# Running the Project
+## Run Script
+The `run.sh` & `run_with_endpoint.sh` scripts can be utilized to run the S3D manually.
 
 It should be noted that this script must be run from the project root folder, or it will not work.
 
-# Docker Compose Files
-The docker-compose.yml file can be used to spin up the PPM as a container, along with instances of kafka and zookeeper.
+### Environment Variables
+The following environment variables must be set in order for the script to run properly.
```

Comment:
```suggestion
The following environment variables must be set in order for the scripts to run properly.
```

Reply:
This line now uses the plural `scripts`.